src/views/Scenario/Scenario.js

```diff
@@ -228,6 +245,30 @@ const Scenario = () => {
     label: scenarioListLabel,
     validationStatus: scenarioValidationStatusLabels,
   };
+  // FOR DEVELOPMENT PURPOSE ONLY, DELETE BEFORE MERGE
+  const lastRun = '2023-03-13T16:25:30.795486885Z';
+  // FOR DEVELOPMENT PURPOSE ONLY, DELETE BEFORE MERGE
+
+  const hasScenarioBeenRun = useMemo(
+    () => currentScenario?.data?.lastRun !== null && currentScenario?.data?.state === 'Successful',
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [currentScenario?.data?.lastRun]
+  );
+  const isDashboardSync = useMemo(
+    () => new Date(currentScenario?.data?.lastUpdate) < new Date(lastRun),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [currentScenario?.data?.lastUpdate]
+  );
+  const getDashboardCardStyle = () => {
+    return hasScenarioBeenRun
+      ? !isDirty && isDashboardSync
+        ? {
+            px: 3,
+            '&:last-child': { pb: 2 },
+          }
+        : { px: 3, bgcolor: 'dashboard.warning', '&:last-child': { pb: 2 } }
```

Comment:
Both objects seem to be very close except for the `bgcolor` key, can we factorize the common style? (maybe we could create both constants above the component definition?)